osbuild/buildroot.py

```diff
@@ -220,6 +220,11 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
         mounts += ["--tmpfs", "/tmp"]
         mounts += ["--bind", self.var, "/var"]
 
+        # Create a usable /var/tmp, see
+        #  https://github.com/osbuild/bootc-image-builder/issues/223
+        os.makedirs(os.path.join(self.var, "tmp"), 0o1777, exist_ok=True)
```

Comment:
Thanks! `self.var` is just a tmpdir() generated from osbuild. I initially just made it a symlink to /tmp but then I just felt uneasy (for no good reason probably) because in general /var/tmp is backed up a real disk and /tmp might be on a tmpfs so I felt like it is worth keeping that semantic. I'm probably overthinking it though.

Reply:
Yes, `/var/tmp` is documented to be a persistent filesystem...by *default*.  But no code involved as part of a build process (whether that's dpkg/rpm/etc) can expect that to happen :smile:  And obviously `/var/tmp` doesn't persist in docker/podman containers by default either.